Source/GNUmakefile

```diff
@@ -357,7 +357,12 @@ GSXibKeyedUnarchiver.m \
 GSXib5KeyedUnarchiver.m \
 GSHelpAttachment.m \
 GSAutoLayoutEngine.m \
-GSAutoLayoutVFLParser.m
+GSAutoLayoutVFLParser.m \
+GSCassowarySolver.m \
+GSCSConstraint.m \
```

Comment:
I've used the prefix 'GSCS' to denote an auxiliary class for the GSCassowarySolver, if a different naming convention is preferred, let me know

Reply:
I think this is perfectly fine.